tests/components/thumbnail-test.js

```diff
@@ -142,4 +142,27 @@ describe('Thumbnail', function () {
       expect(thumbnailTime.length).toBe(1);
     }
   });
+
+  it('tests functions for vr preview', function () {
```

Comment:
Does this unit test test all the changes we've made above? It seems like there should be more unit tests. Please add some more coverage if needed. Thanks!

Reply:
Ok, thank you, but in next PR a code for 360 thumbnail will be changed. So I add more tests in next PR.